common/student/screening.ts

```diff
@@ -5,10 +5,12 @@ import { getLogger } from '../../common/logger/logger';
 import { createRemissionRequest } from '../remission-request';
 import { getTransactionLog } from '../transactionlog';
 import CoCCancelledEvent from '../transactionlog/types/CoCCancelledEvent';
+import { screening_jobstatus_enum, instructor_screening_jobstatus_enum } from '../../graphql/generated';
 
 interface ScreeningInput {
     success: boolean;
     comment?: string;
+    jobStatus?: screening_jobstatus_enum | instructor_screening_jobstatus_enum;
```

Comment:
prisma generates two different generated/enums out of the one entity/JobStatus enum, which I cannot use here for some reason (getting type error). That is why I have this kinda weird or statement for one and the same thing.

Reply:
I recently found out that one can pass an enum name to the Column(...) declaration, in which case there will be only one enum created in the DB, this npx db pull will also only create one enum

Reply:
Ok cool will check it out

Reply:
Ok cool will check it out

Reply:
Ok cool will check it out